### DISCUSSION TITLE
How to parse the binary file of the star catalog?

### DISCUSSION THREAD

**Discussion by tsuiwade** [Stellarium/stellarium#2118] [Q&A] [answered]
I have knowed that the stars catalog file is in \stars\default. 
And how to parse this binary catalog file?

And where did you download this catalog file from?

Thanks!

**Comment by gzotti** [comment, marked as answer]
The catalog is described in the User Guide. Source code is in ZoneArray and related classes. And it is in need of a thorough re-work.

**Reply by tsuiwade** [reply]
Thank you for your answer!
I followed your instructions to check.
It may be different from my understanding.
Is there such a file that includes the RA/Rec magnitude etc. data of Star?

Looking forward to your answer, thank you
